Fix wrong rpath - this should be computed from the dependencies list, in case
they're not in the same prefix...

--- qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt.orig	2019-10-23 19:09:56.000000000 +0200
+++ qt_gui_cpp/src/qt_gui_cpp_sip/CMakeLists.txt	2019-11-27 09:33:33.396472935 +0100
@@ -29,7 +29,7 @@
 # maintain context for different named target
 set(qt_gui_cpp_sip_INCLUDE_DIRS ${qt_gui_cpp_INCLUDE_DIRS} "${CMAKE_CURRENT_SOURCE_DIR}/../../include" ${catkin_INCLUDE_DIRS})
 set(qt_gui_cpp_sip_LIBRARIES ${qt_gui_cpp_LIBRARIES} ${PROJECT_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
-set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} ${CATKIN_DEVEL_PREFIX}/lib)
+set(qt_gui_cpp_sip_LIBRARY_DIRS ${qt_gui_cpp_LIBRARY_DIRS} ${CMAKE_INSTALL_PREFIX}/lib)
 set(qt_gui_cpp_sip_LDFLAGS_OTHER ${qt_gui_cpp_LDFLAGS_OTHER})
 
 find_package(python_qt_binding REQUIRED)
